cores/arduino/CDCSerialClass.cpp

```diff
@@ -73,7 +100,9 @@ void CDCSerialClass::init(const uint32_t dwBaudRate, const uint8_t modeReg)
      * Empty the Rx buffer but don't touch Tx buffer: it is drained by the
      * LMT one way or another */
     _rx_buffer->tail = _rx_buffer->head;
-
+    
+    mailbox_register(6, cdc_mbox_isr);
```

Comment:
Throughput would be vastly improved if we fill all 16 bytes of the mailbox channel, instead of sending 1 message per character. Let's focus on that first, and then see if it's stable enough to use one channel for RX/TX